sdks/wasm/src/binding_support.js

```diff
@@ -1151,9 +1151,9 @@ var BindingSupportLib = {
 
 		var coreObj = BINDING.mono_wasm_get_global()[js_name];
 
-		if (coreObj === null || typeof coreObj === undefined) {
+		if (coreObj === null || typeof coreObj === "undefined") {
```

Comment:
Can you say `coreObj == undefined`?

Reply:
yes, but want to follow the same format as in other places when checking for undefined so more for consistency.  The `typeof` operator is guaranteed to return a string.

Reply:
Also, for consistency, using `===` instead of `==` that will check for strict type.

Reply:
Presumably string compare is slower than object compare, and to be avoided, but I don't know.